app/code/Magento/Catalog/Test/Mftf/ActionGroup/AdminCheckImageRolesSelectedActionGroup.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminCheckImageRolesSelectedActionGroup">
```

Comment:
Would you, please, split this action group into two separate action groups? Like the following example:

- `AdminOpenProductImagesSectionActionGroup`
- `AssertAdminProductImageRolesSelectedActionGroup`

Reply:
I splitted the logic, thank you for suggestions.